### PR TITLE
SEO-179238 WindowsForms Page with Redirect

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan ,
Have changed Page with Redirect links.
Regards,
Yvone.

### CODE REVIEW

File: WindowsForms/Installation/web-installer/how-to-download.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Our 30-day trial can be downloaded in two ways.
    ![Trial and downloads of Syncfusion Essential Studio](images/trial-confirmation.png)
    
 3. With a trial license, only the latest version’s trial installer can be downloaded.
-4. After downloading, the Syncfusion Windows Forms web installer can be unlocked using either the trial unlock key or the Syncfusion registered login credential. More information on generating an unlock key can be found in [this](https://www.syncfusion.com/kb/8069/how-to-generate-unlock-key-for-essentials-studio-products) article.
+4. After downloading, the Syncfusion Windows Forms web installer can be unlocked using either the trial unlock key or the Syncfusion registered login credential. More information on generating an unlock key can be found in [this](https://support.syncfusion.com/kb/article/7053/how-to-generate-unlock-key-for-essentials-studio-products?isInternalRefresh=False) article.
```
Comment (MallikaRK) [inline WindowsForms/Installation/web-installer/how-to-download.md:33]: remove the additional url parameter

File: WindowsForms/Installation/web-installer/how-to-download.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ You should initiate an evaluation if you have already obtained our components th
 
    N> If you've already used the trial products and they haven't expired, you won't be able to start the trial for the same product again.
 
-4. After you've started the trial, go to the [Trials & Downloads](https://www.syncfusion.com/account/manage-trials/downloads) page to get the latest version trial installer. You can generate the [unlock key](https://www.syncfusion.com/kb/8069/how-to-generate-unlock-key-for-essentials-studio-products) and [license key](https://help.syncfusion.com/windowsforms/licensing/how-to-generate) here at any time before the trial period expires. (as shown in below screenshot.)
+4. After you've started the trial, go to the [Trials & Downloads](https://www.syncfusion.com/account/manage-trials/downloads) page to get the latest version trial installer. You can generate the [unlock key](https://support.syncfusion.com/kb/article/7053/how-to-generate-unlock-key-for-essentials-studio-products?isInternalRefresh=False) and [license key](https://help.syncfusion.com/windowsforms/licensing/how-to-generate) here at any time before the trial period expires. (as shown in below screenshot.)
```
Comment (MallikaRK) [inline WindowsForms/Installation/web-installer/how-to-download.md:54]: remove here too.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/Installation/web-installer/how-to-download.md (hunk 1 above)
Comment (Yvone-Atieno) [reply WindowsForms/Installation/web-installer/how-to-download.md]: removed

File: WindowsForms/Installation/web-installer/how-to-download.md (hunk 2 above)
Comment (Yvone-Atieno) [reply WindowsForms/Installation/web-installer/how-to-download.md]: removed

**Review by MallikaRK — APPROVED** [review]
approved

**Pull request merged**